try～catch
0.6～
-----------------------------------------------------------
[ 機能 ]
例外処理を設定します。

[ 書式 ]
try {
  <コマンド>
} catch {
  <例外処理>
}

[ 例 ]
try {
  if (e=1) {
    throw 1;
  }
} catch {
  log("error\n");
}

[ 解説 ]
<コマンド> 内で想定外の動作が発生した場合に throw を記載します。
この場合 catch 以降の <例外処理> を実行します。

